<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputDialog</class>
 <widget class="QDialog" name="InputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>450</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 22pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>450</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 22pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>姓名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>出生日期：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>49</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;隶书&quot;;</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="boyradioButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>男</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">sexGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="girlradioButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>女</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">sexGroup</string>
   </attribute>
  </widget>
  <widget class="QDateEdit" name="birthdaydateEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;Agency FB&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;隶书&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>111</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>学号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>英语：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="englishEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;隶书&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>数学：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mathEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;隶书&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>390</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>语文：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="chineseEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;隶书&quot;;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="sexGroup"/>
 </buttongroups>
</ui>
